it/uk/gov/hmrc/emcstfe/controllers/SubmitAlertOrRejectionControllerIntegrationSpec.scala

```diff
@@ -53,6 +56,7 @@ class SubmitAlertOrRejectionControllerIntegrationSpec extends IntegrationBaseSpe
 
   override def beforeEach(): Unit = {
     disable(SendToEIS)
+    sys.props -= EnableNRS.configName
```

Comment:
It ensures that each test starts in a default state (i.e., when `getFeatureSwitchValue` is called it will always resort to the `application.conf` value not a 'cached' state from a previous test)

Reply:
I will set it to `disable(EnableNRS)` for consistency with other tests, though 😄